### PR TITLE
Fixed Sim Icons, Added Watchdog, fixed error

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Fixed Sim Icons, Added Watchdog, fixed error.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-glove.yml [hunk 1]
```diff
@@ -123,13 +123,28 @@
...
 
+- name: Install the Shadow System Monitor desktop icon for Teleop
+  include_role:
+    name: products/common/web-gui-icon
+  vars:
+    desktop_icon_png: "system_monitor.png"
+    launch_script: "shadow_launcher_system_monitor_exec.sh"
+    local_website_port_var: '8081'
+    desktop_icon_name: "Shadow System Monitor"
+    desktop_icon_path: "Shadow System Monitor"
```
Comment (maxzieba) [inline ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-glove.yml:134]: something is not right here, https://github.com/shadow-robot/aurora/blob/f3bda17c1b4394514a0eff399f7d232ad6b9bd9f/ansible/roles/products/common/web-gui-icon/tasks/main.yml#L24

Comment (BenStarmerSmith) [reply ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-glove.yml:134]: Theres no issue. It works as intended as seen here:
Screenshot from 2022-08-12 09-07-15

Comment (maxzieba) [reply ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-glove.yml:134]: desktop_icon_path is only the name of the icon, the 'order' of how the path is created is bad even though it works

Comment (BenStarmerSmith) [reply ansible/roles/products/hand-e/server/desktop-icons/tasks/hand-and-glove.yml:134]: Its created this way because its moved later on in the process

**Review by maxzieba — APPROVED** [review]

**Review by bdenoun — APPROVED** [review]

**Pull request merged**